recipe/meta.yaml

```diff
@@ -35,8 +40,10 @@ test:
     - tests_random
     - LICENSE
   commands:
+    - pip check
     - rm tests/test-docs.py
-    - pytest tests -k 'not test_fread_from_cmd2 and not test_dt_version'
+    # Skipping test_random_attack because it requires files that are not part of sdist (tests_random/methods)
+    - pytest -vv tests -k 'not test_fread_from_cmd2 and not test_random_attack and not test_dt_version'
```

Comment:
What about test_fread_from_cmd2 and test_dt_version?

Reply:
Addressed. I added a patch to fix the problem with `test_dt_version` and I adjusted the comment to explain why `test_fread_from_cmd2` is also skipped.